rpc/src/v1/types/account_info.rs

```diff
@@ -41,3 +43,19 @@ pub struct HwAccountInfo {
 	/// Device manufacturer.
 	pub manufacturer: String,
 }
+
+/// account derived from a signature
+/// as well as information that tells if it is valid for
+/// the current chain
+#[derive(Debug, Clone, Serialize, PartialEq, Eq)]
+#[serde(rename_all="camelCase")]
+pub struct BasicAccount {
+	/// address of the recovered account
+	pub address: Address,
+	/// public key of the recovered account
+	pub public_key: Public,
+	/// if the signature contains chain replay protection
+	/// and the chain_id encoded within the signature matches the current chain
+	/// this would be true, otherwise false.
```

Comment:
Good docs on this type; would be great if you could fix the line lengths and start all sentences with a capital letter.